Dockerfile

```diff
@@ -11,9 +11,9 @@ RUN echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >
 RUN apt-get update && \
   apt-get install -y google-chrome-beta
 
-RUN pip3 install selenium
+RUN pip3 install selenium --break-system-packages
```

Comment:
Why is this needed?

Reply:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/364effbb-40ba-4f0c-84ce-cc15eea05456">

I didn't find better workaround

Reply:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/364effbb-40ba-4f0c-84ce-cc15eea05456">

I didn't find better workaround